lib/cfg-tree.c

```diff
@@ -1065,6 +1066,38 @@ cfg_tree_compile(CfgTree *self)
   return TRUE;
 }
 
+static gboolean
+are_the_persist_ids_unique(const GPtrArray * const initialized_pipes)
+{
+  GHashTable * pipe_persist_ids = g_hash_table_new(g_str_hash, g_str_equal);
+  gboolean     result           = TRUE;
```

Comment:
Please don't use vertical alignments. It may be your preference but we have to use the coding convention of the software we are working on.